migrator/src/main/java/com/google/phonenumbers/migrator/MigrationJob.java

```diff
@@ -44,15 +46,18 @@
   private final CsvTable<RangeKey> recipesTable;
   private final ImmutableList<MigrationEntry> migrationEntries;
   private final DigitSequence countryCode;
+  private final boolean lenientExport;
```

Comment:
Could you add a comment above this as it isn't obivous when looking at this file what it is.

Reply:
See my above comment about renaming this parameter to be more self-explanatory